src/app/zap-templates/zcl/data-model/silabs/ha.xml

```diff
@@ -348,19 +348,19 @@ limitations under the License.
       <description>
         Move to a specific color temperature.
       </description>
-      <arg name="ColorTemperature" type="INT16U"/>
+      <arg name="ColorTemperatureMireds" type="INT16U"/>
       <arg name="TransitionTime" type="INT16U"/>
       <arg name="OptionsMask" type="BITMAP8"/>
       <arg name="OptionsOverride" type="BITMAP8"/>
     </command>
   </cluster>
   <bitmap name="ColorControlFeature" type="BITMAP32">
     <cluster code="0x0300" />
-    <field name="HueAndSaturation" mask="0x01" />
-    <field name="EnhancedHue" mask="0x02" />
-    <field name="ColorLoop" mask="0x04" />
+    <field name="Hue/Saturation" mask="0x01" />
+    <field name="Enhanced Hue" mask="0x02" />
+    <field name="Color loop" mask="0x04" />
     <field name="XY" mask="0x08" />
-    <field name="ColorTemperature" mask="0x10" />
+    <field name="Color temperature" mask="0x10" />
```

Comment:
Aren't we looking to CamelCasing everything ?
The generator create a ColorTemperature at the end .. i am confused

Reply:
I understand why you changed it. But I am not convinced it should be written like this in the code. The feature should also be CamelCased. Maybe it should be changed the other way around. (In the spec)

Reply:
What I am looking to do is to get to a point where once we start either generating the spec from the XML or scraping the XML from the spec there are no changes involved in either one in the process.

That means completely aligning the XML with the spec now.

Reply:
And yes, we could do that by changing the spec, but I have also been getting pushback on "gratuitous" spec changes, so trying to only change the spec when I feel it really needs to happen.

Reply:
🐫🐫🐫🐫🐫🐫🐫🐫🐫🐫🐫🐫

Reply:
> And yes, we could do that by changing the spec, but I have also been getting pushback on "gratuitous" spec changes, so trying to only change the spec when I feel it really needs to happe

this would be editorial on the spec.. who is giving you a hard time ?

Reply:
This change here makes the XML worse. It means there must be more and more conversion rules for any users of XML/generated code in other contexts to remove spaces or even SLASHES. We have a big problem here in terms of use friendliness. I would expect ALL identifiers in the spec to: 

1. Have no spaces
2. Have no slashes

Reply:
Undid these changes here, and will try to fix them in the spec.